diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/AnnotationNewLineRule.kt

```diff
@@ -70,10 +71,9 @@ class AnnotationNewLineRule : Rule("annotation-new-line") {
             }
 
             if(leftSide) {
-                if (node.treePrev?.isWhiteSpace() == true) {
-                    node.removeChild(node.treePrev)
+                if (node.treeParent.treeParent.treePrev.isWhiteSpace()) {
```

Comment:
This will trigger for every annotation, not only the first one. Also, please add a comment in code explaining why you need to go up two levels (i.e. what is the AST structure here, which nodes are parents of `ANNOTATION_ENTRY`).